src/test/regress/expected/start_stop_metadata_sync.out

```diff
@@ -0,0 +1,306 @@
+CREATE MATERIALIZED VIEW test_matview AS SELECT COUNT(*) FROM distributed_table_3;
+ALTER TABLE distributed_table_4 DROP COLUMN c;
+-- test for hybrid partitioned table (columnar+heap)
+CREATE TABLE events(ts timestamptz, i int, n numeric, s text)
+  PARTITION BY RANGE (ts);
+CREATE TABLE events_2021_jan PARTITION OF events
+  FOR VALUES FROM ('2021-01-01') TO ('2021-02-01');
+CREATE TABLE events_2021_feb PARTITION OF events
+  FOR VALUES FROM ('2021-02-01') TO ('2021-03-01');
+INSERT INTO events SELECT
+    '2021-01-01'::timestamptz + '0.45 seconds'::interval * g,
+    g,
+    g*pi(),
+    'number: ' || g::text
+    FROM generate_series(1,1000) g;
+VACUUM (FREEZE, ANALYZE) events_2021_feb;
+SELECT create_distributed_table('events', 'ts');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT alter_table_set_access_method('events_2021_jan', 'columnar');
+ alter_table_set_access_method
+---------------------------------------------------------------------
+
+(1 row)
+
+VACUUM (FREEZE, ANALYZE) events_2021_jan;
+-- sync metadata
+SELECT start_metadata_sync_to_node('localhost', :worker_1_port);
+ start_metadata_sync_to_node
+---------------------------------------------------------------------
+
+(1 row)
+
+\c - - - :worker_1_port
+SET search_path TO "start_stop_metadata_sync";
+SELECT * FROM distributed_table_1;
+ col | b
+---------------------------------------------------------------------
+(0 rows)
+
+CREATE VIEW test_view AS SELECT COUNT(*) FROM distributed_table_3;
+CREATE MATERIALIZED VIEW test_matview AS SELECT COUNT(*) FROM distributed_table_3;
+SELECT * FROM test_view;
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+SELECT * FROM test_matview;
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+SELECT * FROM pg_dist_partition WHERE logicalrelid::text LIKE 'events%' ORDER BY logicalrelid::text;
+  logicalrelid   | partmethod |                                                          partkey                                                           | colocationid | repmodel
+---------------------------------------------------------------------
+ events          | h          | {VAR :varno 1 :varattno 1 :vartype 1184 :vartypmod -1 :varcollid 0 :varlevelsup 0 :varnoold 1 :varoattno 1 :location -1} |      1390012 | s
+ events_2021_feb | h          | {VAR :varno 1 :varattno 1 :vartype 1184 :vartypmod -1 :varcollid 0 :varlevelsup 0 :varnoold 1 :varoattno 1 :location -1} |      1390012 | s
+ events_2021_jan | h          | {VAR :varno 1 :varattno 1 :vartype 1184 :vartypmod -1 :varcollid 0 :varlevelsup 0 :varnoold 1 :varoattno 1 :location -1} |      1390012 | s
+(3 rows)
+
+SELECT count(*) > 0 FROM pg_dist_node;
+ ?column?
+---------------------------------------------------------------------
+ t
+(1 row)
+
+SELECT count(*) > 0 FROM pg_dist_shard;
```

Comment:
maybe add `WHERE logicalrelid IN (`events`::regclass, `events_2021_feb`::regclass, `events_2021_jan`::regclass )`

Also,

SELECT count(*) FROM pg_dist_shard_placement WHERE shardi  IN (SELECT shardid FROM pg_dist _shard WHERE WHERE logicalrelid IN (`events`::regclass, `events_2021_feb`::regclass, `events_2021_jan`::regclass  


(to all tests we have which returns true)